examples/2d_isometry/cars.rs

```diff
@@ -0,0 +1,138 @@
+use amethyst::assets::{AssetStorage, Loader};
+use amethyst::core::{IsometricTransform, Time};
+use amethyst::ecs::{Component, Join, Read, System, VecStorage, World, WriteStorage};
+use amethyst::prelude::*;
+use amethyst::renderer::{
+    FilterMethod, MaterialTextureSet, PngFormat, SamplerInfo, Sprite, SpriteRender, SpriteSheet,
+    SpriteSheetHandle, Texture, TextureCoordinates, TextureMetadata, Transparent, WrapMode,
+};
+
+use map::UNIT_DIMENSIONS;
+
+const CARS_TEXTURE_ID: u64 = 1;
+
+pub struct Car {
+    top: f32,
+    bottom: f32,
+    speed: f32,
+    up_sprite: usize,
+    down_sprite: usize,
+}
+
+impl Component for Car {
+    type Storage = VecStorage<Self>;
+}
+
+pub fn initialise_cars(world: &mut World, sprite_sheet: SpriteSheetHandle) {
+    let mut iso_transf =
+        IsometricTransform::from_unit_dimensions(UNIT_DIMENSIONS.0, UNIT_DIMENSIONS.1);
+    iso_transf.translation.x = 1.5;
+    iso_transf.translation.y = 1.3;
+    iso_transf.order_priority = 300.0;
+    world
+        .create_entity()
+        .with(iso_transf)
+        .with(Car {
+            top: 2.3,
+            bottom: 0.0,
+            speed: 0.5,
+            up_sprite: 0,
+            down_sprite: 1,
+        })
+        .with(SpriteRender {
+            sprite_sheet,
+            sprite_number: 0,
+            flip_horizontal: false,
+            flip_vertical: false,
+        })
+        .with(Transparent)
+        .build();
+}
+
+pub fn load_cars_sprite_sheet(world: &mut World) -> SpriteSheetHandle {
+    let texture_handle = {
+        let loader = world.read_resource::<Loader>();
+        let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+        loader.load(
+            "texture/vehicles_blue.png",
+            PngFormat,
+            TextureMetadata::default()
+                .with_sampler(SamplerInfo::new(FilterMethod::Trilinear, WrapMode::Border)),
```

Comment:
nit: There is only one mipmap level so bilinear would be fine also

Reply:
Do we care? lol